pypy/doc/install.rst

```diff
@@ -6,7 +6,7 @@ extra packages. The choices for installing the base interpreter are:
 
 - Use conda (x86_64 windows, macOS, linux, arm64 linux)
 - Use your distribution package manager (linux)
-- Use homebew (macOS, but see issue 3697_)
+- Use homebrew (macOS, but see issue 3696_)
```

Comment:
By the way, I wonder if it's still necessary to mention issue 3696 -- as of May 23, 2024, there don't seem to be any problems installing pypy or pypy3 on MacOS Homebrew anymore.

Reply:
thanks, done in 30ded988c20